numba/npyufunc/parfor.py

```diff
@@ -1539,4 +1552,7 @@ def load_range(v):
         only_elem_ptr = builder.gep(rv_arg, [context.get_constant(types.intp, 0)])
         builder.store(builder.load(only_elem_ptr), lowerer.getvar(k))
 
+#    for v in redarrdict.values():
+#        lowerer.lower_inst(ir.Del(v.name, loc=loc))
+
```

Comment:
dead code?

Reply:
This is the code that made me want to look at the PR again.  Want to make sure the leak appears to be gone.  I also had the same question about how to test this and I don't know the answer. @stuartarchibald

Reply:
ah ok, thanks. I'll put working out how to test this on my TODOs, I'm sure we can come up with something, even if we have to write something new it'd probably prove quite useful.

Reply:
Removed those commented lines.